presto-server-rpm/pom.xml

```diff
@@ -67,110 +67,101 @@
-                                </mapping>
-                                <mapping>
-                                    <!-- This should go to just /usr/lib/presto eventually. But that needs modifying
-                                    launcher.py in airlift, to have a configurable option for install_path -->
-                                    <directory>/usr/lib/presto/lib</directory>
-                                    <sources>
-                                        <source>
-                                            <location>${project.build.directory}/${server.tar.package}/lib
-                                            </location>
-                                        </source>
-                                    </sources>
-                                </mapping>
-                                <mapping>
-                                    <directory>/usr/lib/presto/lib/plugin</directory>
-                                    <sources>
-                                        <source>
-                                            <location>
-                                                ${project.build.directory}/${server.tar.package}/plugin
-                                            </location>
-                                        </source>
-                                    </sources>
-                                </mapping>
-                                <mapping>
-                                    <directory>/etc/presto</directory>
-                                    <sources>
-                                        <source>
-                                            <location>src/main/resources/dist/config</location>
-                                        </source>
-                                    </sources>
-                                </mapping>
-                                <mapping>
-                                    <directory>/usr/shared/doc/presto</directory>
-                                    <sources>
-                                        <source>
-                                            <location>${project.build.directory}/${server.tar.package}
-                                            </location>
-                                            <!-- Just include Presto's README for now.-->
+                        <groupId>uk.co.codezen</groupId>
+                        <artifactId>redlinerpm-maven-plugin</artifactId>
+                        <version>1.0</version>
+                        <extensions>true</extensions>
+
+			            <configuration>
+                            <performCheckingForExtraFiles>false</performCheckingForExtraFiles>
+
+                            <packages>
+                                <package>
+                                    <name>presto</name>
+                                    <version>${project.version}</version>
+                                    <release>1.1</release>
```

Comment:
Looks like RPMs have to have release. Removing release gives an RPM name like presto-0.109.SNAPSHOT-1435674331.x86_64.rpm, which changes every time, since the default release for the plugin we use is the UNIX epoch (similarly, the standard Maven RPM plugin specifies release of a SNAPSHOT to be the timestamp, and anything not a SNAPSHOT is 1). I think we should keep it at 1.